content/ngf/overview/resource-validation.md

```diff
@@ -62,9 +62,9 @@ More information on CEL in Kubernetes can be found [here](https://kubernetes.io/
 This step catches the following cases of invalid values:
 
 - Valid values from the Gateway API perspective but not supported by NGINX Gateway Fabric yet. For example, a feature in an HTTPRoute routing rule. For the list of supported features see [Gateway API Compatibility]({{< relref "./gateway-api-compatibility.md" >}}) doc.
-- Valid values from the Gateway API perspective, but invalid for NGINX, because NGINX has stricter validation requirements for certain fields. These values will cause NGINX to fail to reload or operate erroneously.
-- Invalid values (both from the Gateway API and NGINX perspectives) that were not rejected because Step 1 was bypassed. Similar to the previous case, these values will cause NGINX to fail to reload or operate erroneously.
-- Malicious values that inject unrestricted NGINX config into the NGINX configuration (similar to an SQL injection attack).
+- Valid values from the Gateway API perspective, but invalid for NGINX. NGINX has stricter validation requirements for certain fields. These values will cause NGINX to fail to reload or operate erroneously.
+- Invalid values (both from the Gateway API and NGINX perspectives) that were not rejected because Step 1 was bypassed. These values will cause NGINX to fail to reload or operate incorrectly.
+- Malicious values that inject unchecked NGINX config into the NGINX configuration (similar to an SQL injection attack).
```

Comment:
Believe it or not "unrestricted NGINX config" is actually a "technical term"
<img width="750" alt="Screenshot 2025-05-04 at 11 45 13 AM" src="https://github.com/user-attachments/assets/ef52d75b-c575-44ed-a04e-1ea6ff58d838" />
```suggestion
- Malicious values that inject unrestricted NGINX config into the NGINX configuration (similar to an SQL injection attack).
```